### ISSUE TITLE
[Bug]: cann't drag the snippet back to folder from trash

### ISSUE THREAD

**Issue by Benbinbin** [massCodeIO/massCode#210]
### Describe the bug

cann't drag the snippet back to folder from trash

### To reproduce

https://user-images.githubusercontent.com/40909550/183628107-e85ed0f4-f020-4403-a271-2154cd161cb2.mp4

1. Go to `Trash`
2. Click on snippet
3. drag to a folder
4. the snippet cann't drag back to folder, still stay at trash, and see an error 👇  in Developer Tools

```
index.0193768c.js:73 Uncaught (in promise) TypeError: e is not iterable
    at Proxy.getSnippetsByFolderIds (index.0193768c.js:73)
    at Proxy.<anonymous> (index.0193768c.js:8)
    at q (index.0193768c.js:1476)
```

the version of massCode and System

![Snipaste_2022-08-09_18-37-11](https://user-images.githubusercontent.com/40909550/183628454-23b94029-7bd9-4e8b-8aeb-0a407521b668.png)

### Version

3.1.0

### OS

Windows11

### Validations

- [X] Follow our [Code of Conduct](https://github.com/massCodeIO/massCode/blob/master/CODE_OF_CONDUCT.md)
- [X] Check that there isn't [already an issue](https://github.com/massCodeIO/massCode/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/massCodeIO/massCode/discussions).

**Comment by antonreshetov** [comment]
This action is generally not provided, so it is unlikely bug. But thanks for the error itself.

**Comment by Benbinbin** [comment]
@antonreshetov thanks for your answer, I think if massCode support the withdraw deletion action maybe more convenient

**Comment by antonreshetov** [comment]
Closed because it was fixed in #221
